gravitee-apim-e2e/lib/management-webclient-sdk/src/lib/apis/ApplicationsApi.ts

```diff
@@ -2145,14 +2159,77 @@ export class ApplicationsApi extends runtime.BaseAPI {
     }
 
     /**
-     * User must have MANAGEMENT_APPLICATION[READ] and PORTAL_APPLICATION[READ] permission to list applications.
+     * User must have MANAGEMENT_APPLICATION[READ] and PORTAL_APPLICATION[READ] permission to list applications. User must have ORGANIZATION:ADMIN role to list all ARCHIVED applications.
      * List all the applications accessible to authenticated user
      */
     async getApplications(requestParameters: GetApplicationsRequest): Promise<Array<ApplicationEntity>> {
         const response = await this.getApplicationsRaw(requestParameters);
         return await response.value();
     }
 
+    /**
+     * User must have MANAGEMENT_APPLICATION[READ] and PORTAL_APPLICATION[READ] permission to list applications. User must have ORGANIZATION:ADMIN role to list all ARCHIVED applications.
+     * List all the applications accessible to authenticated user
+     */
+    async getApplicationsPagedRaw(requestParameters: GetApplicationsPagedRequest): Promise<runtime.ApiResponse<ApplicationListItemPagedResult>> {
+        if (requestParameters.envId === null || requestParameters.envId === undefined) {
+            throw new runtime.RequiredError('envId','Required parameter requestParameters.envId was null or undefined when calling getApplicationsPaged.');
+        }
+
+        if (requestParameters.orgId === null || requestParameters.orgId === undefined) {
+            throw new runtime.RequiredError('orgId','Required parameter requestParameters.orgId was null or undefined when calling getApplicationsPaged.');
+        }
+
+        const queryParameters: runtime.HTTPQuery = {};
```

Comment:
Juste style: 
for this kind of use case i usually use : 
{
...(isNil(requestParameters.group) ? {group: requestParameters.group}: {} )
ext
}
 to try to reduce this code

Reply:
haha generated ok je sort !